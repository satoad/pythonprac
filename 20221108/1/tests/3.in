a = DivStr("XcDfQWFNJNJDKSNKFJNjnjknfjsnfEasdERTddishifhsiohfishdfosihfosidhfosidhfRTY")
print(* a // 8)
print(a % 11)
print(* a % 10 // 3)
print(a.lower() % 5)
print(* a[1:15] // 10)
print(a % 5 + DivStr() + a % 8)