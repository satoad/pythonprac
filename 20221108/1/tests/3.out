XcDfQWFNJ NJDKSNKFJ Njnjknfjs nfEasdERT ddishifhs iohfishdf osihfosid hfosidhfR
sidhfRTY
f R T
frty
c D f Q W F N J N J
fRTYTY
